From: Michel Normand <normand@linux.vnet.ibm.com>
Subject: libpgm 5.2.122 configure rdtsc checking chg
Date: Fri, 16 Aug 2019 13:20:53 +0200

libpgm 5.2.122 configure rdtsc checking chg

To avoid rdtsc to be enabled  when not required
(for PowerPC ARM) when LTO is enabled; eg openSUSE:
https://build.opensuse.org/package/show/devel:libraries:c_c++/openpgm

Signed-off-by: Michel Normand <normand@linux.vnet.ibm.com>
---
 configure.ac |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: pgm/configure.ac
===================================================================
--- pgm.orig/configure.ac
+++ pgm/configure.ac
@@ -153,7 +153,7 @@ darwin*)
 	AC_MSG_RESULT([no])
 	;;
 *)
-AC_COMPILE_IFELSE(
+AC_LINK_IFELSE(
 	[AC_LANG_PROGRAM(,[[unsigned long lo, hi;
 __asm__ __volatile__ ("rdtsc" : "=a" (lo), "=d" (hi));]])],
 	[AC_MSG_RESULT([yes])
